commit 46003075924bb16b41149e7c12f31e9a6b7a3f15
Author: Kefu Chai <kchai@redhat.com>
Date:   Wed Jan 24 01:13:50 2018 +0800

    mgr: be compatible with py3
    
    so ceph-mgr can be compiled using py3
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/PyModuleRunner.cc b/src/mgr/PyModuleRunner.cc
index 81d74473504..5b98a70279a 100644
--- a/src/mgr/PyModuleRunner.cc
+++ b/src/mgr/PyModuleRunner.cc
@@ -13,7 +13,7 @@
 
 
 // Python.h comes first because otherwise it clobbers ceph's assert
-#include "Python.h"
+#include "PythonCompat.h"
 
 #include "PyModule.h"
 
